WNPRC_EHR/resources/schemas/dbscripts/postgresql/wnprc-21.006-21.007.sql

```diff
@@ -0,0 +1,5 @@
+alter table wnprc.animal_requests add column pregnantanimalsrequiredterminfant varchar(100);
```

Comment:
Add if exists clause, this will prevent from having problems updating rsehr in the future.